tests/src/Functional/AccessTest.php

```diff
@@ -29,7 +29,6 @@ class AccessTest extends BrowserTestBase {
      */
     protected static $modules = [
         'node',
-        'media',
```

Comment:
@AaronGilMartinez Alright so we rely on indirect dependency from 'collabora_online'.
This makes me wonder if we need 'node' at all.